robots/alexk_lcr/INSTALLATION.md

```diff
@@ -23,17 +23,17 @@ cargo install dora-cli
 Now you're ready to run Rust dataflow applications! We decided to only make Python dataflow for AlexK Low Cost Robot, so
 you may need to setup your Python environment:
 
-- Install Python 3.8 or later by following the instructions at [Python](https://www.python.org/downloads/).
+- Install Python 3.12 or later by following the instructions at [Python](https://www.python.org/downloads/).
 - Clone this repository by running the following command:
 
 ```bash
 git clone https://github.com/dora-rs/dora-lerobot
 ```
 
-- Open a bash terminal and navigate to the root of the submodule `alexk_lcr` in the repository.
+- Open a bash terminal and navigate to the repository by running the following command:
 
 ```bash
-cd dora-lerobot/alexk_lcr
+cd dora-dora_lerobot
```

Comment:
```suggestion
cd dora-lerobot
```
Maybe?

Reply:
Thank you, I hadn't noticed all these issues that my Refactoring tool caused :'(, I'll go back over it and fix everything.

Reply:
No props, it happens :)